src/commands/generate_buildpack_matrix/command.rs

```diff
@@ -1,34 +1,70 @@
-use crate::buildpacks::{find_releasable_buildpacks, read_image_repository_metadata};
+use crate::buildpacks::{
+    find_releasable_buildpacks, read_buildpack_descriptor, read_image_repository_metadata,
+};
 use crate::commands::generate_buildpack_matrix::errors::Error;
+use crate::commands::resolve_path;
 use crate::github::actions;
 use clap::Parser;
-use libcnb_package::{
-    get_buildpack_target_dir, read_buildpack_data, BuildpackData, GenericMetadata,
+use libcnb_data::buildpack::{BuildpackDescriptor, BuildpackId};
+use libcnb_package::output::{
+    create_packaged_buildpack_dir_resolver, default_buildpack_directory_name,
 };
+use libcnb_package::CargoProfile;
 use std::collections::{BTreeMap, HashSet};
-use std::path::Path;
+use std::path::{Path, PathBuf};
 
 type Result<T> = std::result::Result<T, Error>;
 
 #[derive(Parser, Debug)]
 #[command(author, version, about = "Generates a JSON list of buildpack information for each buildpack detected", long_about = None)]
-pub(crate) struct GenerateBuildpackMatrixArgs;
+pub(crate) struct GenerateBuildpackMatrixArgs {
+    #[arg(long)]
+    pub(crate) working_dir: Option<PathBuf>,
```

Comment:
This PR adds a new `--working-dir` argument to most of the CLI commands (for some it's a rename of `--path`), however nothing seems to set it (e.g. from the GitHub Actions workflow), so it seems unused?

Whilst for end-user tooling (such as `cargo libcnb`) having an arg to override the working directory might be useful, for the CNB release automation in this repo, we're the only consumers, so if we're not using an arg then we might as well drop it. (We can always add it back later if needed.)

Dropping these unused args would also simplify a lot of the debate over the path resolving functionality discussed in:
https://github.com/heroku/languages-github-actions/pull/143#discussion_r1347103266

Reply:
After speaking with Manuel, he agreed, so I've gone ahead and addressed this in c7c3dd4ce258a21bc0641dcabd87dd21bdb978b0 to speed things up.

Between that commit and the `--release` removal one, the PR has now reduced in size from `+527 −605` to `+490 −601`.